.changelog/2743.txt

```diff
@@ -0,0 +1,5 @@
+```release-note:improvement
+control-plane: Changed the container ordering in connect-inject to insert consul-dataplane
```

Comment:
**nitpick:** Please remove line breaks. Your editor probably added them automatically (mine does this all the time).